meson.build

```diff
@@ -0,0 +1,1570 @@
+project('mpv',
+        'c',
+        license: ['GPL2+', 'LGPL2.1+'],
+        version: '0.33',
+        default_options: [
+            'buildtype=debugoptimized',
+            'c_std=c11',
+            'warning_level=1',
+        ]
+)
+
+cc = meson.get_compiler('c')
+
+flags = ['-D_ISOC99_SOURCE', '-D_GNU_SOURCE',
+         '-D_FILE_OFFSET_BITS=64']
+link_flags = []
+
+test_flags = ['-Werror=implicit-function-declaration',
+              '-Wno-error=deprecated-declarations',
+              '-Wno-error=unused-function',
+              '-Wempty-body',
+              '-Wdisabled-optimization',
+              '-Wstrict-prototypes',
+              '-Wno-format-zero-length',
+              '-Werror=format-security',
+              '-Wno-redundant-decls',
+              '-Wvla',
+              '-Wno-format-truncation',
+              '-Wimplicit-fallthrough',
+              '-fno-math-errno']
+
+foreach flag: test_flags
```

Comment:
Could this loop be replaced by get_supported_arguments (since 0.43.0)? I guess the following should work:
```
flags += get_supported_arguments(test_flags)
```

Reply:
Yup that definitely works.